webextensions/browser-compat-data.json

```diff
@@ -1199,6 +1199,9 @@
     }, 
     "TemplateType": {
       "Chrome": {
+        "notes": [
+          "For the 'basic' type, 'iconUrl' is mandatory."
+        ],
```

Comment:
Only in create(), I think? I don't think TemplateType is the right place to document this, and it is already covered in the main docs for https://developer.mozilla.org/en-US/Add-ons/WebExtensions/API/notifications/NotificationOptions